docs/Examples.md

```diff
@@ -725,7 +725,7 @@ There are two ways to configure `pass4Symmkey` with an External LM:
   ```
 - Setup the above decrypted plain-text [`pass4Symmkey`](PasswordManagement.md#pass4Symmkey) in the global secret object(Note: The `pass4Symmkey` would be stored in a base64 encoded format). For details see [updating global kubernetes secret object](#updating-global-kubernetes-secret-object)
 
-### Configuring license_master_url:
+### Configuring license_manager_url:
```

Comment:
Probably should leave this until this env variable name changes.

Reply:
That's a good point. We can roll them back for now.

Reply:
Done.